tests/polyfill.ts

```diff
@@ -1,4 +1,4 @@
-import { neonConfig } from '../../dist/npm';
+import { neonConfig } from '..';
```

Comment:
I wonder if this works?
If it does, it might be nice cause it'd also verify `package.json`'s fields, right?

```suggestion
import { neonConfig } from '@neondatabase/serverless';
```

Reply:
Yep, that works fine (because of `"@neondatabase/serverless": "file:."` in `"devDependencies"`).

Reply:
I wonder if it makes it import the source `.ts` file or the file pointed at in `package.json`'s `main`/`module`/whatever field?

If it's not pointing at the `.ts` one then it's probably better to switch it back? So Vitest can run in watch mode over the changing sources

Reply:
No, we need to test the built and bundled code, because that's where the shims come in.